tools/vendor/vendor-drift.py

```
@@ -0,0 +1,416 @@
+    raise SystemExit(2)
+
+
+def fail(message):
+    print("vendor-drift: %s" % message, file=sys.stderr)
+    raise SystemExit(1)
+
+
+def run(argv, cwd=None):
+    """Run a command, returning (rc, stdout, stderr). Never raises on rc."""
+    try:
+        proc = subprocess.run(argv, cwd=cwd, capture_output=True, text=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)